opentracing/dapperish/trivial_recorder.go

```diff
@@ -9,25 +8,29 @@ import (
 )
 
 type TrivialRecorder struct {
-	tags map[string]string
+	processName string
+	tags        map[string]string
 }
 
-func NewTrivialRecorder() *TrivialRecorder {
+func NewTrivialRecorder(processName string) *TrivialRecorder {
```

Comment:
note that the recorder interface is effectively forced into accepting a process name as a param (this is a good thing IMO)